rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/qos.hpp

```diff
@@ -27,6 +27,8 @@
 
 #include "rmw_fastrtps_shared_cpp/visibility_control.h"
 
+#include "rosidl_runtime_c/type_hash.h"
```

Comment:
It looks like `rmw_fastrtps_shared_cpp` doesn't directly depend on `rosidl_runtime_c` right now.  So we need to add it to the package.xml .